modules/data-loading/partials/load-part1-intro-and-schema.adoc

```diff
@@ -1,10 +1,8 @@
-After you have xref:gsql-ref:ddl-and-loading:defining-a-graph-schema.adoc[defined a graph schema], you can create a loading job, specify your data sources, and run the job to load data.
+After you have xref:gsql-ref:ddl-and-loading:defining-a-graph-schema.adoc[defined a graph schema] you can create a loading job, specify your data sources, and run the job to load data.
 
-The steps are similar whether you are loading from local files, from cloud storage, or any of the other supported sources.
-We will call out whether a particular step is common for all loading or specific to a data source or loading mode.
+The steps for loading from local files, cloud storage, or any other supported sources are similar. We will distinguish where steps are common for all loading jobs, or are specific to a particular data source or loading mode.
```

Comment:
```suggestion
The steps for loading from local files, cloud storage, or any other supported sources are similar.
We will call out whether a particular step is common for all loading or specific to a data source or loading mode.
```
Rejecting the changes to the second sentence. "Call out" is more semantically appropriate than "distinguish". We are calling attention.  Also, it is better asciidoc style for each sentence to be on a separate line, to simplify revisions.